src/monitor/Monitor.ts

```diff
@@ -596,6 +599,157 @@ export class Monitor {
     }
   }
 
+  async checkSpokePoolRunningBalances(): Promise<void> {
+    // We define a custom format function since we do not want the same precision that `convertFromWei` gives us.
+    const formatWei = (weiVal: string, decimals: number) => {
+      if (weiVal === "0") {
+        return "0";
+      }
+      const formatFunction = createFormatFunction(1, 4, false, decimals);
+      return formatFunction(weiVal);
+    };
```

Comment:
RE: https://github.com/across-protocol/relayer/pull/1868/files#r1809105128

Given what we're trying to do maybe we can condense this?

```suggestion
    const formatWei = (weiVal: string, decimals: number) =>
        weiVal === "0" ? "0" :
        createFormatFunction(1, 4, false, decimals)(weiVal);
```